UI/Pages/Game.razor

```diff
@@ -10,18 +10,32 @@
 <PageTitle>TicTacGo Game</PageTitle>
 
 <div class="pt-20">
-  @if (_host is not null && _host.HasWon)
-  {
-    <WinnerScreen Winner="@_host"/>
-  }
-  else if (_guest is not null && _guest.HasWon)
-  {
-    <WinnerScreen Winner="@_guest"/>
-  }
-  else
-  {
-    if (_gameCanStart)
+    @if (_gameCanStart)
     {
+      @if (_host.HasWon)
```

Comment:
Check if `_host` is not `null` please, like this:

```
_host != null && _host.HasWon
```

Or the new way, which is:

```
_host is {HasWon: true }
```

That does the same thing.

Reply:
will do, thank you